src/Reader/Reader.php

```diff
@@ -546,8 +548,10 @@ public static function detectType($feed, $specOnly = false)
 
     /**
      * Set plugin manager for use with Extensions
+     *
+     * @return void
      */
-    public static function setExtensionManager(ExtensionManagerInterface $extensionManager)
+    public static function setExtensionManager(ExtensionManagerInterface $extensionManager): void
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.